### PR TITLE
convert windmill min timestamp to beam min timestamp

### PR DESCRIPTION
**Pull request by Naireen**
**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by Naireen** [comment]
Run Java PreCommit

**Comment by Naireen** [comment]
Run Java PreCommit

File: runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/WindmillTimerInternals.java [hunk 1]
```diff
@@ -351,12 +356,25 @@ public static TimerData windmillTimerToTimerData(
...
       try {
         outputTimestamp =
             new Instant(
                 VarInt.decodeLong(
                     tag.substring(outputTimestampStart, outputTimestampEnd).newInput()));
+        System.out.println(
+            "parsed string"
+                + String.valueOf(
+                    tag.substring(outputTimestampStart, outputTimestampEnd).newInput()));
+        if (outputTimestamp.isBefore(BoundedWindow.TIMESTAMP_MIN_VALUE)) {
```
Comment (robertwb) [inline runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/WindmillTimerInternals.java:372]: Can we re-use the code above?

File: runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/WindmillTimeUtils.java [hunk 2]
```diff
@@ -42,13 +42,23 @@ public class WindmillTimeUtils {
    * round timestamps down and output watermarks up.
    */
   public static Instant windmillToHarnessTimestamp(long timestampUs) {
+    System.out.println("check if need to bound timestamp: " + String.valueOf(timestampUs));
```
[the remark this answers is not in the document]
Comment (Naireen) [reply runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/WindmillTimeUtils.java]: Done

**Comment by Naireen** [comment]
Run Java PreCommit

**Comment by scwhittle** [comment]
LGTM, is there a github issue that should be referenced?

**Comment by aaltay** [comment]
Merging this. Tests are passing and @scwhittle reviewed it.

**Pull request merged**